README.MD

```diff
@@ -0,0 +1,39 @@
+# MERN Project
+This is a project to develop a funcionl app that allows blogging, registering users, chatting and other things. There will be a deployment in Heroku of the app.
+
+## System Requirements:
```

Comment:
In my opinion, this could be in a different place/file. Check [this](https://github.com/Xhoria-RB/SocialShare-backend/projects) and let me know what you think about putting this specs in a project.

Reply:
By now let's leave it there and i'll start a project in GitHub later on.